app/javascript/components/pages/MySearchIndex.js

```diff
@@ -2,28 +2,28 @@ import React from "react"
 import { Link } from "react-router-dom"
 
 class MySearchIndex extends React.Component {
-    render() {
-        return (
-            <React.Fragment>
-                <h3>Saved TICities</h3>
-                <div id="search-index-body">
-                    {this.props.searches.map((search, index) => {
-                        return (
-                            <div key={index} className="searches-card">
-                                <h3>{search.city1}</h3>
-                                <h3>{search.city2}</h3>
-                                <h3>{search.city3}</h3>
-                                <br />
-                                <Link to={`/searchresults/${search.id}`} className="button">
-                                    More Info
-                                </Link>
-                            </div>
-                        )
-                    })}
-                </div>
-            </React.Fragment>
-        );
-    }
```

Comment:
Y'all should make sure your editors are set to have tabs as 2 spaces rather than 4. 

Also, consider setting up this JS formatter called `prettier` in your project, adding the extension in your editors, and then enabling format on save. That way, you'll have consistent formatting across PRs. 
https://prettier.io/docs/en/install.html

Not a blocker for this one though.

Reply:
added prettier extension to atom/vscode